scripts/globals/abilities/devotion.lua

```diff
@@ -40,7 +40,7 @@ function onUseAbility(player,target,ability)
     healMP = utils.clamp(healMP, 0,target:getMaxMP() - target:getMP())
     -- printf("Devotion MP Healed: %d", healMP)
 
-    player:delHP(damageHP)
+    player:takeDamage(damageHP)
```

Comment:
Needs to take stoneskin into account.
`Stoneskin can mitigate the amount of HP sacrificed. However, the ability does not lose potency this way.`